.env.template

```diff
@@ -0,0 +1,5 @@
+# It will force the user to add an e-mail for this project, before committing.
+COMMIT_MAIL=my.commit@mail.com
+
+# (https://app.zeplin.io/profile/developer)
+DEVELOPER_ZEPLIN_ACCESS_TOKEN=TODO
```

Comment:
What does this have to do with this topic / pull request ?

Reply:
https://github.com/db-ui/base/blob/feat-zeplin-token-generation/scripts/zeplin-styleguide.js#L7

We need this as an access token to fetch the json from zeplin